### DISCUSSION TITLE
Install Docker Engine

### DISCUSSION THREAD

**Discussion by MelindaPozzi** [l3atbc-datadog/pushkin#118] [General] [answered]
Hi!

I have Windows 10 and I have followed all the steps to install Pushkin and dependencies, but I have an issue when I try to install Docker Engine (on Ubuntu 18.04 LTS). When I want to check that Docker Engine is installed correctly (by running sudo docker run hello-world), I got this error message: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

What to I need to do to solve this problem?

Thank you in advance!
Mélinda Pozzi

**Comment by restmegan** [comment, marked as answer]
Try running `docker info` to check if Docker is running. If it isn't, run `sudo dockerd` to start Docker then try `sudo docker run hello-world` again and let us know if that worked!

**Reply by MelindaPozzi** [reply]
Thank you for the answer! I still get an error message. This is what I get when I run sudo dockerd:

![Docker Engine](https://user-images.githubusercontent.com/33163405/89815211-aa524980-db44-11ea-8b99-610c36b2ad59.JPG)
![Docker Engine2](https://user-images.githubusercontent.com/33163405/89815221-b211ee00-db44-11ea-9adc-a75e2285bcbb.JPG)

**Reply by parkerrobbins** [reply]
Hi, @MelindaPozzi! Sometimes I have trouble with my ECS instance being super slow. Because this is a `timeout` issue, I'm wondering if you've tried logging out of your instance and logging back in? Sometimes that does it for me.

**Reply by MelindaPozzi** [reply]
Ah yes, I stopped my instance and I forgot to restart it before installing Docker Engine. Now it works! Thank you very much for your help!

**Reply by parkerrobbins** [reply]
Thanks for your reply! Glad you've got it working!
